src/audio/dai-zephyr.c

```diff
@@ -1257,8 +1258,10 @@ static int dai_copy(struct comp_dev *dev)
 
 	/* limit bytes per copy to one period for the whole pipeline
 	 * in order to avoid high load spike
+	 * if FAST_MODE is enabled, then copy as much data as possible
```

Comment:
Two questions: 
a) should the FAST_MODE be the default on all host copiers? 
b) is the comment on line 1259 related to load spike relevant? Why would copying more data lead to a change in load? downstream modules still have their period size, deadlines and cpc definition, so what would happen if the host copier copied more data?

Reply:
a) No, it should be disabled by default. That feature is enabled to satisfy some specific modules requirements for fw encoders/decoders where there is no abovious obs/ibs limitations of size. 
b) That feature allows to freely trasfer nontypical amount of data to feed for example decoder module or transfer data out of encoder implemented in fw. In my opinion there is no risk of copying too much data at once in encoder/decoder scenario. it is still limited upwards by size of internal copier buffer which is 2*ibs(obs).